classes/blocks/class-articles.php

```diff
@@ -275,7 +170,7 @@ public function load_more() {
 	 *
 	 * @return array
 	 */
-	private function populate_post_items( $posts ) {
+	public function populate_post_items( $posts ) {
```

Comment:
These can still be private as they're only used in the class. What I meant was adding `static` keyword to it :slightly_smiling_face:

Reply:
Otherwise the usages look like this :stuck_out_tongue: 
![Screenshot from 2020-07-21 10-36-28](https://user-images.githubusercontent.com/7604138/88032370-2527d600-cb3e-11ea-93fd-f16c7d1d09aa.png)

Reply:
🙈 Sorry I guess I read your comment too fast!